docs/source/PyMOLplugin.rst

```diff
@@ -0,0 +1,233 @@
+PyMOL plugin
+============
+
+This plugin provides tools for identification, classification and analysis of
+conserved water molecules and their networks from molecular dynamics
+simulation trajectories.
+
+The main interface consists of the following sections:
+
+Input Options
+-------------
+
+Input trajectories **MUST** be aligned and centered before analysis. The
+plugin will not perform any alignment or centering. If the system is
+centered on the protein, it usually works well. However, in instances
+where the area of interest is close to the edge of the simulation box,
+system should be centered on the position of the center of mass of the area of
+interest.
+
+**Selection Input Type**: Choose the type of input you want to provide. Options include:
+
+.. tabs:: 
+
+  .. tab:: **PyMOL Selection**
+
+    A PyMOL selection is first exported to a PDB file and waters are
+    extracted using water selection panel using MDAnalysis.
+
+    **PyMOL Selection**: Enter the selection string which contains
+    waters (can contain other elements as well) in your PyMOL window. We
+    recommend selection contains system as well as waters of interest.
+
+    **Test Input**: Click this button to test the provided input.
+
+  .. tab:: **From Files**
+      
+    Provide trajectory and topology files. 
+
+    **Trajectory and Topology Files**: Browse and select the appropriate
+    files for your analysis. For list of supported file types check
+    MDAnalysis documentation `here
+    <https://www.mdanalysis.org/docs/documentation_pages/coordinates/init.html#supported-file-formats>`_.
```

Comment:
This should be the right link: https://docs.mdanalysis.org/stable/documentation_pages/coordinates/init.html#supported-coordinate-formats